src/environment.rs

```diff
@@ -252,11 +270,17 @@ impl EnvironmentMap {
     pub fn write_features(&self, mut w: &fs::File) -> io::Result<()> {
         use io::Write;
 
-        let mut features = self.get_override_var("FEATURES").unwrap_or_else(|| {
-            self.filter_map_keys(|k| k.strip_prefix("CARGO_FEATURE_"))
-                .map(|f| f.to_owned())
-                .collect::<Vec<_>>()
-        });
+        let mut features: Vec<String> = self
+            .get_override_var("FEATURES")
+            .unwrap_or_else(|| {
+                self.get_var::<Vec<String>>("CARGO_CFG_FEATURE")
```

Comment:
Why is `get_var` required here, instead of just `self.get()` (like [here](https://github.com/lukaslueg/built/blob/4c71c96441e3fb3fd98fd4f36081448ebe0a96e8/src/environment.rs#L151-L154))?

Reply:
It looked a bit messy using `.get()` and then `parse_from_env` here in order to get a `Vec<String>`. 

happy to switch to that if you prefer it.

 also happy for you to take over to adjust this PR to how you prefer it.